app/src/main/java/com/xa/xpensauditor/EmailSender.java

```diff
@@ -91,6 +107,11 @@ private Boolean sendEmailUpdate(String message) {
         return true;
     }
 
+    /**
+     * Override for the default task related setup.
+     * @param voids - Leave empty. No inputs expected beyond creation of the object.
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.